jerry-core/vm/vm.c

```diff
@@ -1432,14 +1194,154 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           *stack_top_p++ = left_value;
           continue;
         }
+        case VM_OC_CREATE_BINDING:
+        {
+#if !ENABLED (JERRY_ES2015)
+          if (opcode == CBC_CREATE_LET
+              || opcode == CBC_CREATE_CONST
+              || opcode == CBC_CREATE_LOCAL)
+          {
+            continue;
```

Comment:
We should replace this code with an assertion here. Only CBC_CREATE_VAR is allowed.